.testcaferc.json

```
@@ -5,5 +5,10 @@
     "takeOnFails": false
   },
   "skipJsErrors": true,
-  "stopOnFirstFail": true
-}
+  "stopOnFirstFail": true,
+  "clientScripts": [
+    {
+      "module": "axe-core/axe.min.js"
```

```suggestion
      "module": "axe-core/axe.js"
```

Might help with debugging.

---

I wonder if the minified script runs quicker which might add up in the long run if we start using it more. But probably not.

---

That's fair. If we end up needing to debug something, we can switch to `axe.js`.